docs/tutorials/integrations/nginx/tls/tls.conf

```diff
@@ -0,0 +1,39 @@
+[req]
+default_bits = 2048
+prompt = no
+default_md = sha256
+req_extensions = req_ext
+distinguished_name = dn
+x509_extensions = v3_ca # The extentions to add to the self signed cert
+req_extensions  = v3_req
+x509_extensions = usr_cert
+
+[ dn ]
+C=US
+ST=Wisconsin
+L=Madison
+O=Onyx
+OU=CyberArk
```

Comment:
Shouldn't these be swapped? O is organization, OU is organizational unit. Just saying... ;)